dashboard/modules/serve/sdk.py

```diff
@@ -11,10 +11,11 @@
 from ray.dashboard.modules.dashboard_sdk import SubmissionClient
 
 
-DEPLOY_PATH = "/api/serve/deployments/"
+DEPLOY_APP_PATH = "/api/serve/applications/"
+# TODO(sihanwang) Changed to INFO_APPLICATION_PATH
 INFO_PATH = "/api/serve/deployments/"
-STATUS_PATH = "/api/serve/deployments/status"
-DELETE_PATH = "/api/serve/deployments/"
+STATUS_APP_PATH = "/api/serve/applications/status/"
+DELETE_APP_PATH = "/api/serve/application/{app_name}"
```

Comment:
ideally we'd support something like this:
`PUT /api/serve/applications/` -> deploy the full config
`GET /api/serve/applications/` -> returns the full list of app statuses
`DELETE /api/serve/applications/` -> shuts down all the apps
`GET /api/serve/applications/{app_name}` -> returns the single app status
`PUT /api/serve/applications/{app_name}` -> updates the single app
`DELETE /api/serve/applications/{app_name}` -> deletes the single app

Reply:
at a minimum for kuberay / the CLI we need the top-level `PUT/GET/DELETE /api/serve/applications/`, the per-app ones are optional